docs/resources/uberbank_example1.ini

```diff
@@ -134,46 +139,39 @@ approximant = SEOBNRv2_ROM_DoubleSpin_HI
 iterative-match-df-max = 8.0
 cache-waveforms =
 
+mtotal-max = 100
+mratio-min = 1
+mratio-max = 97.989
+
+aligned-spin =
+bh-spin-min = -${workflow|max-bh-spin}
+bh-spin-max = ${workflow|max-bh-spin}
+ns-spin-min = -${workflow|max-ns-spin}
+ns-spin-max = ${workflow|max-ns-spin}
+ns-bh-boundary-mass = ${workflow|nsbh-boundary}
+
 [sbank-bbh]
 ; These options are sent to all sbank "BBH" jobs
 
 ; Mass/spin parameter options
```

Comment:
So part of the idea of "BBH" and "final" was so that "BBH" would fill in the part of parameter space where signals have already been found and then final would fill everything else. "BBH" used a higher minimal match to ensure a dense coverage in the "BBH" region, with only a small increase in template count.